### ISSUE TITLE
LDAP auth

### ISSUE THREAD

**Issue by arvanus** [FirebirdSQL/firebird#7326]
Hi, is there an available LDAP auth method? 
I see that there is Windows Security auth, but would know more about using AD/LDAP in a Linux environment
Thanks

**Comment by aafemt** [comment]
No. You are welcome to create an auth plugin for it. An example is in examples/extauth dir.
